components/typography/typography.css

```diff
@@ -0,0 +1,56 @@
+h1,
+.h1 {
+  font-size: 3.2rem;
+  margin: 10px 0;
+}
+h2,
+.h1 {
```

Comment:
I believe this is supposed to be h2 class, typo

Reply:
thanks for pointing it out, resolved